CHANGES.md

```diff
@@ -1,3 +1,25 @@
+# 2.15.0 - 22 Jul 2024
+
+## Features
+
+* Add support for Pebble check-failed and check-recovered events (#1281)
+
+## Fixes
+
+* Use temp dir for secret data (#1290)
```

Comment:
I think this could probably be a bit more detailed? Maybe:

```suggestion
* Pass secret data to Juju via files, rather than as command-line values (#1290)
```